highlight/highlight
# Attach error feedback to error alerts

## Summary



This PR overloads the error alert to also include feedback sent via the [error boundary component](https://www.highlight.io/docs/getting-started/client-sdk/reactjs).
Previously, there was an alert named "Session Feedback" that could be set up to trigger this type of alert. However, it was confusing as it was also overloaded with the the now deprecated `feedbackWidget` config (#5413). This PR removes that alert type entirely.

_Note: per this [comment](https://github.com/highlight/highlight/issues/5115#issuecomment-1538507868), we can only link this feedback to a session. We currently can't figure out what error it belongs to._

## How did you test this change?



* Create a new error alert with slack + discord configured.

* Issue feedback via the error boundary component:
![Screenshot 2023-06-05 at 10 35 17 AM](https://github.com/highlight/highlight/assets/58678/a41833c0-262a-4355-ae6c-ddf45c3278ac)

* Observe in Slack that we get the error alert and then the error boundary feedback:
![Screenshot 2023-06-05 at 10 36 42 AM](https://github.com/highlight/highlight/assets/58678/6c51a9bf-1c6b-48dc-ac8d-a169e073299a)


* Observe in Discord that we get the error alert and then the error boundary feedback:
![Screenshot 2023-06-05 at 10 36 55 AM](https://github.com/highlight/highlight/assets/58678/1b1241d6-a0df-49b6-9b60-096a20ee0af2)


* Observe that both link out to the commentID
![Screenshot_2023-06-05_at_10_35_48_AM](https://github.com/highlight/highlight/assets/58678/173dbac2-e6e7-42a6-9d25-6a65f472189d)




## Are there any deployment considerations?



N/A
